ckanext/datatablesview/plugin.py

```diff
@@ -22,6 +22,14 @@ class DataTablesView(p.SingletonPlugin):
     p.implements(p.IConfigurer, inherit=True)
     p.implements(p.IResourceView, inherit=True)
     p.implements(p.IBlueprint)
+    p.implements(p.ITemplateHelpers, inherit=True)
+
+    # ITemplateHelpers
+
+    def get_helpers(self):
+        return{
+            'encode_datatables_request_filters': helpers.encode_datatables_request_filters
```

Comment:
@JVickery-TBS This sounds like a valuable helper for other view plugins as well. Shall we rename them to `encode/decode_view_request_filters` move it to `ckan/lib/helpers.py` so it's available to everybody?